kategory/src/test/kotlin/kategory/data/OptionTTest.kt

```diff
@@ -29,6 +30,14 @@ class OptionTTest : UnitSpec() {
                 Eq.any(),
                 Eq.any()))
 
+        testLaws(FunctorFilterLaws.laws(
+                OptionT.applicative(Option),
+                OptionT.functorFilter(),
+                object : Eq<HK<OptionTKindPartial<OptionHK>, Int>>{
+                    override fun eqv(a: HK<OptionTKindPartial<OptionHK>, Int>, b: HK<OptionTKindPartial<OptionHK>, Int>): Boolean =
+                            a.ev().value == b.ev().value
```

Comment:
We are doing the same for other laws being tested on this class. I've changed it now to use the already created `OptionTFIdEq` instance (on top part of this test class).